opba-protocols/sandboxes/xs2a-sandbox/src/test/resources/logback-test.xml

```diff
@@ -1,10 +1,10 @@
 <configuration>
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
-            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{50} - %msg%n</pattern>
+            <pattern>%d{HH:mm:ss.SSS} %highlight(%-5level) %cyan(%logger{36}) - %msg %n</pattern>
         </encoder>
     </appender>
-    <root level="warn">
+    <root level="info">
```

Comment:
This increases log amount excessively in Travis builds. Please keep WARN level in tests